ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/http/StripCorsHeadersHandler.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.core.http;
+
+import static io.vertx.core.http.HttpHeaders.ACCESS_CONTROL_ALLOW_HEADERS;
+import static io.vertx.core.http.HttpHeaders.ACCESS_CONTROL_ALLOW_METHODS;
+import static io.vertx.core.http.HttpHeaders.ORIGIN;
+
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import io.vertx.core.Handler;
+import io.vertx.ext.web.RoutingContext;
+
+public class StripCorsHeadersHandler implements Handler<RoutingContext> {
+
+  private static final List<CharSequence> headersToRemove =
```

Comment:
capitalise headersToRemove since it's a constant

Reply:
done